superset/views/core.py

```diff
@@ -2395,7 +2395,7 @@ def validate_sql_json(
         schema = request.form.get("schema") or None
         template_params = json.loads(request.form.get("templateParams") or "{}")
 
-        if len(template_params) > 0:
+        if template_params is not None and len(template_params) > 0:
```

Comment:
Nit, since in Python an empty list/dict is false you can simplify to:

```suggestion
        if template_params:
```

Reply:
thanks! Sorry, I just saw this note after I merged. I can clean it up separately.